client/src/views/CreateAccount.tsx

```diff
@@ -1,10 +1,29 @@
+import { userInfo } from 'os';
 import React from 'react';
+import { useForm } from 'react-hook-form';
 import { Link } from 'react-router-dom';
 import { registerUser } from '../services';
 
+type RegisterUser = {
+  firstName: string;
+  lastName: string;
+  email: string;
+  password: string;
+};
+
 function CreateAccount() {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<RegisterUser>();
+
+  const onSubmit = (data: RegisterUser) => {
+    registerUser(data);
```

Comment:
what's the point of having a function that strictly passes data to another function. just pass registerUser into handleSubmit.